client/src/components/custom-button/custom-button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
created a custom-button components that can be re-used in the future when more button features are needed